app/views/books/index.html.erb

```diff
@@ -32,14 +32,14 @@
         <td><%= book.is_pdf %></td>
         <td><%= book.quantity %></td>
         <td><%= book.price %></td>
-        <td><%= link_to 'Show', book %></td>
-        <td><%= link_to 'Edit', edit_book_path(book) %></td>
-        <td><%= link_to 'Destroy', book, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td><%= link_to t(:show, scope: [:default]), book %></td>
+        <td><%= link_to t(:edit, scope: [:default]), edit_book_path(book) %></td>
+        <td><%= link_to t(:destroy, scope: [:default]), book, method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
localization in the confirm message is missing

Reply:
yes I forgot it